apps/voting/app/src/components/VoteDetail/VoteInfoBoxes.js

```diff
@@ -0,0 +1,193 @@
+import React from 'react'
+import { useNetwork } from '@aragon/api-react'
+import {
+  Box,
+  GU,
+  Help,
+  IconTime,
+  Timer,
+  TransactionBadge,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import { format } from 'date-fns'
+import SummaryBar from '../SummaryBar'
+import VoteStatus from '../VoteStatus'
+import { round, safeDiv } from '../../math-utils'
+import { getQuorumProgress } from '../../vote-utils'
+
+const formatDate = date => `${format(date, 'do MMM yy, HH:mm')}`
+
+function VoteInfoBoxes({ vote }) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const compact = layoutName === 'small'
+  const { numData } = vote
+  const { minAcceptQuorum, supportRequired, yea, nay } = numData
+  const quorumProgress = getQuorumProgress(vote)
+  const totalVotes = yea + nay
+  const votesYeaVotersSize = safeDiv(yea, totalVotes)
+  const votesNayVotersSize = safeDiv(nay, totalVotes)
+
+  return (
+    <div
+      css={`
+        margin-top: ${2 * GU}px;
+        display: grid;
+        grid-template-columns: ${compact ? 'auto' : 'auto auto auto'};
+        grid-template-rows: ${compact ? 'auto auto auto' : 'auto'};
```

Comment:
```suggestion
        grid-auto-flow: ${compact ? 'row' : 'column'};
```
This should also work if you prefer :)